app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt

```diff
@@ -130,7 +132,8 @@ class NewProfileFragment : Fragment() {
                     smin = smin,
                     ehr = ehr,
                     emin = emin,
-                    d = daySelected.toJson(days)
+                    d = daySelected.toJson(days),
+                    c = col
```

Comment:
Why declare a variable col when it is only being used once and can be directly assigned to c?

Reply:
Ok, I will directly assign this